examples/parallel_profiling.py

```diff
@@ -0,0 +1,142 @@
+"""
+================================================
+Memory and CPU consumption using joblib.Parallel
+================================================
+
+This example illustrates some important aspects to keep in mind when using
+:class:`joblib.Parallel`. We will focus on different patterns resulting to the
+same outcomes but which trigger different internal behaviors. Those differences
+will impact the memory consumption and computation time.
+
+To illustrate those differences, the moving average from a large array will be
+computed. Therefore, the average can be computed in parallel at different
+location of the signal.
+
+"""
+from __future__ import division
+
+print(__doc__)
```

Comment:
How about we don't print the docstring? Most people look at those examples online anyhow. It would make the examples simpler.